ios/app/src/ViewController.m

```diff
@@ -78,6 +87,36 @@ - (void)conferenceFailed:(NSDictionary *)data {
 
 - (void)conferenceJoined:(NSDictionary *)data {
     [self _onJitsiMeetViewDelegateEvent:@"CONFERENCE_JOINED" withData:data];
+
+    // Register a NSUserActivity for this conference so it can be invoked as a
+    // Siri shortcut. This is only supported in iOS >= 12.
+#ifdef __IPHONE_12_0
+    if (@available(iOS 12.0, *)) {
```

Comment:
The avid reader may wonder why these 2 similar checks are needed. The former (the ifdef) is needed at build time, to make sure we can build the app with Xcode versions without the iOS 12 SDK, and the latter (the if available) is needed at runtime, so devices not running older iOS versions can run the app but not that part of the code.